README.md

```diff
@@ -323,6 +324,45 @@ All inputs are optional as they may be specified in the Atlas configuration file
    For example: `'{"var1": "value1", "var2": "value2"}'`.
 * `working-directory` - The working directory to run from.  Defaults to project root.
 
+
+### `ariga/atlas-action/migrate/autorebase`
+
+Resolve conflicts in the migration directory by rebasing the current branch's migrations on top of the rebase branch migration directory.
+
+#### Inputs
+
+All inputs are optional
+
+* `rebase-branch` - The branch to rebase on. By default: `main`.
```

Comment:
This is not needed I think. You should use the `GITHUB_BASE_REF` instead (_The name of the base ref or target branch of the pull request in a workflow run._).